### ISSUE TITLE
Adding a change/edit button

### ISSUE THREAD

**Issue by nikhaararoraa** [sk66641/Random-Disco-Light-Simulator#182]
Add a change/edit button to allow modifications while the disco simulator is running. This will enhance functionality and user control.
please assign this to me.

**Comment by bheemajayashreee** [comment]
please assign this issue to me

**Comment by thevijayshankersharma** [comment]
@sk66641 Assign this too if he is inactive.

**Comment by sk66641** [comment]
@nikhaararoraa 
Could you please give an update over this issue as it has been pending for over a week?

**Comment by dhruvishah85** [comment]
@sk66641 Hii can you please assign this to me

**Comment by sk66641** [comment]
@bheemajayashreee 
Want to work on this issue?

**Comment by aditya-bhaumik** [comment]
@sk66641 can i work on this issue if he/she is inactive

**Comment by thevijayshankersharma** [comment]
@sk66641 i will do.

**Comment by dhruvishah85** [comment]
> @thevijayshankersharma or @dhruvishah85 Want to work on this issue or this issue will be assigned to @aditya-bhaumik

Yes i will

**Comment by aditya-bhaumik** [comment]
@sk66641 is this issue still open??

**Comment by sk66641** [comment]
@aditya-bhaumik 
yes, this issue is supposed to add edit button which can edit inputs like number of colors, time interval & view, want to work?

**Comment by aditya-bhaumik** [comment]
@sk66641 Yes, I would like to try adding this feature. No one else is working on this, right?
